addons/full-import/README.md

```diff
@@ -0,0 +1,85 @@
+
+This is a quick readme to describe the export/import process. We should formalize this in our official documentation.
+
+== Assumptions and limitations
+
+ * You can export on any PacketFence version above 10.3.0
+ * You can import on any PacketFence version above 11.0.0
+ * The import process needs to be runned on a standalone server. Restoring directly to clusters is currently unsupported
+  * NOTE: Once you restored to your standalone server, you can make it a cluster by joining other machines to it and creating your cluster.conf but this is relatively advanced and out of scope of this document
+ * Restoring on a fresh install of PacketFence is recommended although restoring on an existing instance can work but your milleage may vary
+ * This process will only restore the files that can be edited via the admin interface which include:
+  * Standard configuration files in /usr/local/pf/conf/*.conf
+  * Connection profiles html templates in /usr/local/pf/html/captive-portal/profile-templates/
```

Comment:
How about if someone uploaded a jpeg/png file (or something else) to be used on his portal?

Reply:
This reminds me I need to grab the file for the `logo` variable of profiles.conf 

But for files referenced in the HTML templates, its pretty hard to track them accurately

I believe that users can use a static/ directory inside their profile-templates/$ID directory which will be migrated through this process.

It may be a bit complex to track assets that are referenced in the HTML template since they won't contain the full path and may be a bit tricky to extract

Reply:
I think we can limit ourself to the logo in profiles.conf for now